recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -211,117 +124,21 @@ this.PreferenceExperiments = {
       preferenceBranchType,
     };
 
-    const prevPrefType = Services.prefs.getPrefType(preferenceName);
-    const givenPrefType = PREFERENCE_TYPE_MAP[preferenceType];
+    // Validation will throw
```

Comment:
@erikrose has convinced me that punctuation is only required for complete sentences.